testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergHiveMetadataListing.java

```diff
@@ -62,7 +64,10 @@ public void testTableListing()
                     .containsOnly(
                             row("iceberg_table1"),
                             row("iceberg_materialized_view"),
-                            row(storageTable));
+                            row(storageTable),
+                            row("iceberg_view"),
+                            // iceberg shows hive views
```

Comment:
"hive view" can be understood as coral-supported hive view. is this what you mean?

```
// Iceberg connector supports views created via Hive connector
```

Reply:
Of course not. Thanks for suggestion.